emloop/cli/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .ls import list_train_dirs
-from .train import train
+
```

Comment:
Is this necessary? We should not modify the API just for the tests. By the way, we can avoid name conflicts just by renaming the module from `train.py` to e.g. `train_fn.py`.

Anyway, it should be consistent for all the commands.